src/Flame/Types.purs

```diff
@@ -69,5 +91,16 @@ data NodeData message =
 
 derive instance nodeDataFunctor :: Functor NodeData
 
+instance showNodeData :: Show (NodeData message) where
+        show (Attribute name val) = "(Attribute " <> show name <> " " <> show val <> ")"
+        show (Property name val) = "(Property " <> show name <> " " <> show val <> ")"
+        show _ = ""
```

Comment:
In both `Show` instances, do you think it could result in better diffs if the strings were not quoted? For example, in a test case for `HE.div [HA.id "1", HA.disabled true, HA.onClick "Test", HA.maxlength 23, HA.createProperty "oi" "ola a todos"]`

`Error: expected "(Node \"div\" [(Property \"id\" \"1\"),(Property \"disabled\" \"true\"),(Property \"maxlength\" \"23\"), (Property \"oi\" \"ola a todo\")] [(Text \"This is my text\")])", got "(Node \"div\" [(Property \"id\" \"1\"),(Property \"disabled\" \"true\"),(Property \"maxlength\" \"23\"),(Property \"oi\" \"ola a todos\")] [(Text \"This is my text\")])"`

vs

`Error: expected "(Node div [(Property id 1),(Property disabled true),(Property maxlength 23), (Property oi ola a todo)] [(Text This is my text)])", got "(Node div [(Property id 1),(Property disabled true),(Property maxlength 23),(Property oi ola a todos)] [(Text This is my text)])"`

Reply:
Sounds reasonable. I'll fix that.